Gems/ROS2/Code/Include/ROS2/Camera/CameraSensor.h

```diff
@@ -10,7 +10,7 @@
 #include <Atom/Feature/Utils/FrameCaptureBus.h>
 #include <AzCore/std/containers/span.h>
 
-#include "CameraPublishers.h"
+#include "../Source/Camera/CameraPublishers.h"
```

Comment:
This is not good - an API header should only include other API headers. I am not sure there is a reason to make the CameraSensor.h a part of the interface.

Reply:
Lets discuss this